pallets/rewards/src/migrations.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
```

Comment:
I've found quite useful to place migrations under a file with a custom name (as `v1.rs` or so), later if in a new PR we remove an old migration and create a new one, there is not an awful git diff, and it becomes easy to review. Not sure if I explain the situation well 😅

Reply:
I get what you are saying but I am not sure I support this. Wouldn't it suffice to push the code into a `mod` inside `migrations.rs` and still guarantee an easy-to-follow git diff in the future?

This particular migration though should actually stick around because we need to run it for every `rewards` instance we spawn.

Reply:
> Wouldn't it suffice to push the code into a mod inside migrations.rs and still guarantee an easy-to-follow git diff in the future?

If you do not remove the previous migration when adding the new one, the diff will be ok, but if you remove it, git will think you're instead modifying the previous one.

Reply:
But does that also apply if we keep them in separate scopes via `mod v1 { ... } mod v2 { ... }` inside the same file?

Reply:
git doesn't know about "scopes", and it will think that you've changed `v1` by `v2`, and will mix every line inside of those blocks.

Reply:
Anyways, nothing mandatory here at all

Reply:
I will follow your proposal! More readable git diffs are definitely worth the little extra work :)

Reply:
To avoid redundant files for the mod.rs, I'm adding them like [this](https://github.com/centrifuge/centrifuge-chain/blob/bd4a96bf6e9bb6606f6b285cfe0493b7415a8734/pallets/loans-ref/src/lib.rs#L40) (in case it can be useful)